src/azure-cli/azure/cli/command_modules/acs/_params.py

```diff
@@ -381,6 +381,7 @@ def load_arguments(self, _):
         c.argument('path', options_list=['--file', '-f'], type=file_type, completer=FilesCompleter(),
                    default=os.path.join(os.path.expanduser('~'), '.kube', 'config'))
         c.argument('public_fqdn', default=False, action='store_true')
+        c.argument('credential_format', type=str, options_list=['--format'], validator=validate_credential_format)
```

Comment:
default value?

Reply:
default value will be `None` as defined in the function definition https://github.com/Azure/azure-cli/blob/0c5bdce72b95facbce1795eb656983790c59652a/src/azure-cli/azure/cli/command_modules/acs/custom.py#L2203